net.go

```diff
@@ -205,11 +205,35 @@ func (m *Memberlist) streamListen() {
 	}
 }
 
+// EnsureCanConnect return the IP from a RemoteAddress
+// return error if this client must not connect
+func (m *Memberlist) EnsureCanConnect(from net.Addr) error {
```

Comment:
May I suggest to rename this to EnsureIsAllowed() instead? 'can connect' can also imply that it checks for whether it would succeed. 'isAllowed' would be less ambiguous I think.

Reply:
DONE